.github/workflows/build-images.yaml

```diff
@@ -4,12 +4,28 @@ on:
   push:
     branches: ['*']
     tags: ['*']
+  workflow_call:
+  workflow_dispatch:
+    inputs:
+      authorinoOperatorBundleVersion:
+        description: Authorino Operator bundle version
+        required: true
+        default: latest
+      limitadorOperatorBundleVersion:
+        description: Limitador Operator bundle version
+        required: true
```

Comment:
Why are these vars required when they are fetched in update-store-dependencies?

Reply:
Those are required if triggered by _workflow_dispatch_, say you want to build a specific bundle with a unique combination of tags, you'll need to provide each one. The ones fetched from _action variables_ are used when a _workflow_call_ was triggered.